src/lib/label/label.component.ts

```diff
@@ -14,7 +17,10 @@ export class PlexLabelComponent {
     @Input() titulo: string;
     @Input() tituloBold = true;
     @Input() subtitulo: string;
-    @Input() size: 'sm' | 'md' | 'lg' = 'md';
+    @Input() size: 'sm' | 'md' | 'lg' | 'xl' = 'md';
+    @Input() icon: string;
+    @Input() direction: 'column' | 'row' = 'row';
+    @Input() type: 'success' | 'info' | 'warning' | 'danger' | 'default';
```

Comment:
Se podría hacer que por defecto tenga el valor 'default' no? así hace honor a su nombre :joy: 
```suggestion
    @Input() type: 'success' | 'info' | 'warning' | 'danger' | 'default' = 'default';
```

Reply:
Ja.. Dale. Gracias Andrews!